### PR TITLE
Fix #224 Provision fails after rebooting the machine

### LINKED ISSUE #224: Provision fails after rebooting the machine
From clean environment, if you do:

```
sudo vagrant up
sudo vagrant halt
sudo vagrant up
sudo vagrant provision
```

You'll get an error about already existing `openshift/origin` image.

I think line [69](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/components/centos/centos-openshift-setup/Vagrantfile#L69) doesn't do it's job as it effectively checks `:latest` image although the next command pulls `:v1.1.1` image.

### PR DESCRIPTION
**Pull request by praveenkumar**
Fix: #224 
This patch also carry updated openshift vagrant file.

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
LGTM. However how to make sure this does not break existing setup of ADB version prior to 1.7

**Comment by praveenkumar** [comment]
We have to add in document because since this is first time ADB going to use `openshift` with systemctl. We can create a dist which can be pointed to doc stating if you are using adb-box older than 1.7 then use this vagrant file.

**Comment by LalatenduMohanty** [comment]
@praveenkumar , May be we should have the old copy of the Vagrantfile in the same directory for some time with appropriate name along with some documentation. 
Thoughts @bexelbie @navidshaikh

**Comment by navidshaikh** [comment]
@LalatenduMohanty : I suggest we do not keep the copies of Vagrantfile for a provider. If ADB 1.7 has openshift provided as systemd service, we should ask users to update the box.

We are working on few enhancements in service-manager plugin which should make the enabling / starting a provider in the box with just one line in Vagrantfile.

**Comment by LalatenduMohanty** [comment]
The other way would be creating a branch for every release and point to the branch in in the readme. Similar to what Kubernetes github project does.

**Comment by navidshaikh** [comment]
@LalatenduMohanty : 

> The other way would be creating a branch for every release and point to the branch in in the readme. Similar to what Kubernetes github project does.

+1

**Comment by LalatenduMohanty** [comment]
@navidshaikh, ideally yes. But I dont want to miss on users who might continue using 1.6.0 and somehow they need the old Vagrantfile.

File: components/centos/centos-openshift-setup/Vagrantfile [hunk 1]
```diff
@@ -5,29 +5,18 @@
 DOCKER_REGISTRY="docker.io"
 
 # The name of the OpenShift image.
-ORIGIN_IMAGE_NAME="openshift/origin:v1.1.1"
+ORIGIN_IMAGE_NAME="openshift/origin"
```
Comment (bexelbie) [inline components/centos/centos-openshift-setup/Vagrantfile:8]: Do we need to allow this value here if we hard code origin for the sub-components below?

Alternately, lines 56-59 should use this tag.

**Comment by bexelbie** [comment]
@LalatenduMohanty 

> The other way would be creating a branch for every release and point to the branch in in the readme. Similar to what Kubernetes github project does.

The users can just go to the tag for the release - there is no need to create branches, imho.

**Comment by LalatenduMohanty** [comment]
@bexelbie , I could not find a way to view the repository for a particular tag using the web browser for the github project. I might not be aware of the method too. Can you point me to somewhere as an example

File: components/centos/centos-openshift-setup/Vagrantfile (hunk 1 above)
Comment (LalatenduMohanty) [inline components/centos/centos-openshift-setup/Vagrantfile:8]: @bexelbie , I think we should fix that.

**Comment by bexelbie** [comment]
@LalatenduMohanty got to the branch switcher and click the tiny word tags next to branches below the input box.
ala https://github.com/projectatomic/adb-atomic-developer-bundle/tree/v1.6.0

**Comment by LalatenduMohanty** [comment]
@bexelbie Cool, works for me

**Comment by LalatenduMohanty** [comment]
LGTM

**Pull request merged**